src/sql/src/parsers/create_parser.rs

```diff
@@ -399,7 +405,7 @@ impl<'a> ParserContext<'a> {
     pub fn parse_column_def(&mut self) -> std::result::Result<ColumnDef, ParserError> {
         let parser = &mut self.parser;
 
-        let name = parser.parse_identifier()?;
+        let name = Self::canonicalize_identifier(parser.parse_identifier()?);
```

Comment:
The method has already called canonicalize_identifier upon returning. Do we still need to call canonicalize_identifier here?